mkosi/install.py

```diff
@@ -61,5 +61,6 @@ def copy_path(
         f"--preserve=mode,timestamps,links,xattr{',ownership' if preserve_owner else ''}",
         "--no-target-directory",
         "--reflink=auto",
+        "--sparse=always",
```

Comment:
Out of curiosity, why is this preferable to the default `auto` behaviour?

Reply:
This was a leftover from something else, dropped the commit